exercises/01-composite-ui/after/Divergent.Sales.Data/Repositories/IOrderRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Divergent.Sales.Data.Repositories
+{
+    using Models;
+
+    public interface IOrderRepository
+    {
+        Task<List<Order>> Orders();
```

Comment:
Why is this Task<T> ?  There's nothing awaitable and can be List<Order>

Reply:
I followed the pattern in the other projects... shall we change all of them maybe?

Reply:
Better to be aligned with the other projects in this PR. We can change all of them in another PR if we decide to.